website/docs/releases/2024/v2024.next.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Release 2024.next
+slug: "/releases/2024.next"
+---
+
+:::::note
+2024.next has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
+To try out the release candidate, replace your Docker image tag with the latest release candidate number, such as 2024.next.0-rc1. You can find the latest one in [the latest releases on GitHub](https://github.com/goauthentik/authentik/releases). If you don't find any, it means we haven't released one yet.
+:::::
+
+## Breaking changes
+
+### PostgreSQL minimum supported version upgrade
+
+authentik now requires PostgreSQL version 14 or later. We recommend upgrading to the latest version if you are running an older version.
+
+The provided Helm chart defaults to PostgreSQL 15. If you are using the Helm chart with the default values, no action is required.
+
+The provided docker-compose file was updated with PostgreSQL 16. You can follow the procedure below to upgrade.
+
+#### docker-compose PostgreSQL upgrade instructions
```

Comment:
I think like we did with the postgres upgrade in K8s, these steps should be on a separate file that can be linked to, like https://github.com/goauthentik/authentik/blob/main/website/docs/troubleshooting/postgres/upgrade_kubernetes.md

Reply:
Oh, haha, I just saw this @BeryJu . Here's my documentation opinion:

Content that is transitory typically belongs either in Rel Notes or a KB... not in regular docs (because it will soon be outdated). My vote is that if this is a breaking change for some users, and it won't be needed by the majority of users past the release of 2024.6, then let's leave it in Rel Notes.

However, if there will be users discovering for many more months to come that they need to upgrade to either 15 or 16, let's move it to Troubleshooting as Jens suggests.

It just could cause our Troubleshooting section to become quite crowded if we always add Upgrade info there. Ah, I think I will [eventually] finally create a new doc just about Upgrading, and that will solve the problem. ;-)

Reply:
> Content that is transitory typically belongs either in Rel Notes or a KB... not in regular docs (because it will soon be outdated). My vote is that if this is a breaking change for some users, and it won't be needed by the majority of users past the release of 2024.6, then let's leave it in Rel Notes.

Agreed, but those upgrade instructions are very unlikely to change for the next time users need to upgrade postgresql, so it makes sense to have them on hand somewhere to link to in the future